src/config.go

```diff
@@ -55,3 +62,16 @@ func (c *Config) loadEnvConfig() *Config {
 
 	return c
 }
+
+func isDockerComposeVersion2(config yamlConfig) bool {
+	var ok bool
+	_, ok = getMapVal(config, "version").(string)
+	if !ok {
+		return false
+	}
+	_, ok = getMapVal(config, "services").(yamlConfig)
+	if !ok {
+		return false
+	}
+	return true
+}
```

Comment:
If both `version` and `services` are present in the docker-compose file, then consider it's using version 2.